client/src/components/register/Register.js

```diff
@@ -39,9 +39,10 @@ function Register(props) {
         axios
           .post('/api/users/login', user)
           .then(res => {
-            //received token
-            console.log(res.data);
-            props.set_LoggedUser(true);
+            localStorage.setItem('authToken', res.data.authToken);
+            localStorage.setItem('userId', res.data.user.id);
+            localStorage.setItem('loggedIn', true);
+            props.onChange(Math.random());
```

Comment:
What's `props.onChange(Math.random());` for?

Reply:
@a-w-m I pass through the `triggerrerender` state as onChange in the App.js to force a rerender.  I wouldn't be able to play with the code until like 9pm EST
The Math.random() just changes the state by making it a random number